src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
@hseok-oh 
>  I think it's better to merge add_subdirectory(src/coreclr/hosts) with Windows and Linux environment, by excluding add_subdirectory(src/coreclr/hosts) at root/CMakeLists.txt

That is not possible. The reason this is included from the root CMakeLists.txt separate is that the Unix hosts are native Unix apps that are not built on top of PAL and so they need to include Unix platform header files. This is something that's disabled later in the root CMakeFiles.txt since none of the runtime code except the PAL is allowed to include these headers.